kde-misc/tellico/metadata.xml

```diff
@@ -12,5 +12,7 @@
 	</longdescription>
 	<use>
 		<flag name="discid">Add support for reading disc IDs from audio CDs</flag>
+		<flag name="yaz">Add optional support for the Z39.50 Protocol
```

Comment:
Let's go with tellico description: 'Add support for searching Z39.50 databases'

Reply:
Ok, I'll update once more people review other changes.

Reply:
Done.